content/docs/reference/buildpack-api.md

```diff
@@ -144,6 +137,89 @@ version = "1.0"
 id = "io.buildpacks.stacks.bionic"
 ```
 
+### Schema
+
+The schema is as follows:
+   
+- **`api`** _(string, required, current: `0.2`)_\
+    The Buildpack API version the buildpack adheres to. Used to ensure [compatibility](#api-compatibility) against
+    the [lifecycle][lifecycle].
+    
+    > Not to be confused with Cloud Foundry or Heroku buildpack versions. This version pertains to the interface
+    > between the [buildpack][buildpack] and the [lifecycle][lifecycle] of Cloud Native Buildpacks.
+
+- **`buildpack`** _(required)_\
```

Comment:
This `buildpack` section is missing `name` (https://github.com/buildpack/spec/blob/master/buildpack.md#buildpacktoml-toml)

Reply:
Good 👀! Added.